usaspending_api/etl/es_etl_helpers.py

```diff
@@ -194,18 +194,24 @@
     "total_covid_outlay",
 ]
 
-UPDATE_DATE_SQL = " AND update_date >= '{}'"
+UPDATE_DATE_SQL = "update_date >= '{}'"
```

Comment:
If not providing the `update_date` sql snippet for the below SQL results in an error now, is there value keeping this string defined separately?